src/nunit-gui/Model/ITestModel.cs

```diff
@@ -73,6 +73,10 @@ public interface ITestModel : IServiceLocator, System.IDisposable
 
         #region Properties
 
+        CommandLineOptions Options { get; }
```

Comment:
The options can be both applied from gui and commandline? I would go with something more generic like RunOptions or just Options unless we need more option-classes?